handlers.js

```diff
@@ -299,6 +299,16 @@ module.exports = function(port) {
             )}
         );
 
+        if (port.bus.config.outOfBandAuthentication &&
```

Comment:
this configuration should not reside in `bus.config`, instead it should be in `port.config`